docs/02-about-us/working-groups/readme.md

```diff
@@ -0,0 +1,9 @@
+# Working Groups
+
+CivicActions maintains [a set of working groups](working-groups) that nurture specific 
+initiatives within the organization.
+
+# Group Listing
+
+* [Documentation](documentation.md)
```

Comment:
If we have a CONTRIBUTING.md and a docs-governance.md page, do we need a page documenting the documentation group too?  Should be able to fit in one of the above?  It also seems like a guild strong candidate for guild like governance.